src/credentials/tests/CHIPCert_test_vectors.cpp

```diff
@@ -169,6 +169,43 @@ CHIP_ERROR GetTestCertPubkey(uint8_t certType, const uint8_t ** certPubkey, uint
     return err;
 }
 
+CHIP_ERROR GetTestCertAKID(uint8_t certType, ByteSpan & akid)
```

Comment:
A more scalable approach is to have a utility in CHIPCryptoPAL (or some other module in src/credentials) that can extract SKID or AKID from a DER certificate. That would reduce the need to separately keep track of the AKID. This could be done in a follow-up. You already add `ExtractAKIDFromX509Cert` in this PR.